src/style.css

```diff
@@ -1,3 +1,74 @@
+@import 'https://fonts.googleapis.com/css?family=Roboto:500&display=swap';
+
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+}
+
+ul {
+  list-style: none;
+  padding: 0;
+  margin: 0;
+}
+
+a {
+  text-decoration: none;
+}
+
+.navigation__wrapper {
+  display: flex;
+  padding: 0 50px;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.navigation__logo {
+  align-self: center;
+  justify-content: center;
+  height: 40px;
+}
+
+.navigation__list {
+  display: flex;
+}
+
+.navigation__item:not(:first-child) {
+  margin-left: 20px;
+}
+
+.navigation__link {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 14px;
+  text-transform: uppercase;
+  text-align: center;
+  display: flex;
+  align-items: center;
+  color: #000;
+  height: 60px;
+}
+
+.navigation__link:hover {
+  color: #00acdc;
+}
+
+.navigation__link:active {
+  color: #00acdc;
+  box-shadow: 0 4px 0 #00acdc;
```

Comment:
CSS Variable is used for a blue color
![image](https://github.com/user-attachments/assets/4fd753a4-4ea1-46a8-9454-52032564fcaa)